imports/server/bugsnag.ts

```diff
@@ -22,7 +22,12 @@ if (apiKey) {
     ],
     onError: (event) => {
       if (Fiber.current) {
```

Comment:
If we try/catch the `Meteor.user()` call, is there any point in the `Fiber.current` check?